tests/content/entity/model/modelReaders/gltfReader/glbTestSuite/buggy/test.js

```diff
@@ -1,45 +1,45 @@
-Script.include(PATH_TO_THE_REPO_PATH_UTILS_FILE);
-var nitpick = createNitpick(Script.resolvePath("."));
-
-nitpick.perform("Read GLTF model", Script.resolvePath("."), "secondary", function(testType) {
-    var assetsRootPath = nitpick.getAssetsRootPath();
-    var LIFETIME = 60.0;
-    var position = nitpick.getOriginFrame();
-
-    Script.include(nitpick.getUtilsRootPath() + "test_stage.js");
-
-    var initData = {
-        flags : {
-            hasKeyLight: true,
-            hasAmbientLight: true,
-            hasKeyLightShadow: true,
-        },
-        originFrame: nitpick.getOriginFrame()
-    };
-    var createdEntities = setupStage(initData);
-
-    var testEntity = Entities.addEntity({
-        lifetime: LIFETIME,
-        type: "Model",
-        // https://github.com/KhronosGroup/glTF-Sample-Models/tree/master/2.0/Buggy
-        modelURL: assetsRootPath + 'models/gltf_models/glb/Buggy.glb',
-        position: Vec3.sum(position, {x: 0.65, y: 1.0, z: -3.5 }),
-        dimensions: Vec3.multiply(50.0, {x: 0.0198, y: 0.0195, z: 0.0202}),
-        rotation: Quat.fromPitchYawRollDegrees(-90.0, -25.0, 0.0),
-        visible: true,
-        userData: JSON.stringify({ grabbableKey: { grabbable: false } })
-    });
-
-    createdEntities.push(testEntity);
-
-    nitpick.addStepSnapshot("Buggy.glb Model is visible");
-
-    nitpick.addStep("Clean up after test", function () {
-        for (var i = 0; i < createdEntities.length; i++) {
-            Entities.deleteEntity(createdEntities[i]);
-        }
-    });
-
-    var result = nitpick.runTest(testType);
-});
+if (typeof PATH_TO_THE_REPO_PATH_UTILS_FILE === 'undefined') PATH_TO_THE_REPO_PATH_UTILS_FILE = "https://raw.githubusercontent.com/highfidelity/hifi_tests/master/tests/utils/branchUtils.js";
+Script.include(PATH_TO_THE_REPO_PATH_UTILS_FILE);
+var nitpick = createNitpick(Script.resolvePath("."));
+
+nitpick.perform("Read GLTF model", Script.resolvePath("."), "secondary", function(testType) {
+    var assetsRootPath = nitpick.getAssetsRootPath();
+    var LIFETIME = 60.0;
+    var position = nitpick.getOriginFrame();
+
+    Script.include(nitpick.getUtilsRootPath() + "test_stage.js");
+
+    var initData = {
+        flags : {
+            hasKeyLight: true,
+            hasAmbientLight: true,
+            hasKeyLightShadow: true,
+        },
+        originFrame: nitpick.getOriginFrame()
+    };
+    var createdEntities = setupStage(initData);
+
+    var testEntity = Entities.addEntity({
+        lifetime: LIFETIME,
+        type: "Model",
+        // https://github.com/KhronosGroup/glTF-Sample-Models/tree/master/2.0/Buggy
+        modelURL: assetsRootPath + 'models/gltf_models/glb/Buggy.glb',
+        position: Vec3.sum(position, {x: 0.25, y: 0.75, z: -1.8 }),
+        dimensions: { x: 1.0, y: 0.5, z: 1.0 },

```

Comment:
Shouldn't dimensions be left undefined and then a scale applied afterward, so we can also test if the scale is being read correctly from the GLTF file?

Reply:
Fixed.